apps/user_ldap/lib/User_Proxy.php

```diff
@@ -431,4 +436,12 @@ public function getDisabledUserList(?int $limit = null, int $offset = 0, string
 			)
 		);
 	}
+
+	protected function treatRemnantsAsDisabled(): bool {
+		static $setting;
+		if (!$setting) {
+			$setting = $this->appConfig->getValueBool('user_ldap', 'markRemnantsAsDisabled', false);
+		}
+		return $setting;
```

Comment:
```suggestion
		return $this->appConfig->getValueBool('user_ldap', 'markRemnantsAsDisabled', false);
```

I would tend to avoid this kind of micro-optimisation which makes code complicated and will bite us later.
If you must, then cache in a property, not in a static var.

Reply:
I think I just wanted to save the function call :shrug: